src/libstd/fs.rs

```diff
@@ -934,7 +934,10 @@ impl Permissions {
     #[stable(feature = "rust1", since = "1.0.0")]
     pub fn readonly(&self) -> bool { self.0.readonly() }
 
-    /// Modifies the readonly flag for this set of permissions.
+    /// Modifies the readonly flag for this set of permissions. If the
+    /// `readonly` argument is `true`, using the resulting `Permission` will
+    /// make the file unwritable. Conversely, if it's is `false`, using the
+    /// resulting `Permission` will make the file writable.
```

Comment:
This all feels pretty wordy, if anyone has suggestions to improve the flow here, would love to hear!